hack/godep-save.sh

```diff
@@ -164,8 +163,13 @@ fork-with-fake-packages github.com/docker/docker \
 # kubernetes tests, we have to extract the missing test dependencies and run godep-save again
 # until we converge. Because we rsync kubernetes itself above, two iterations should be enough.
 MISSING_TEST_DEPS=""
+ALL_DIRS="$(os::util::list_go_src_files | cut -d '/' -f 1-2 | sort -u | grep -v "^./cmd"|grep -v doc.go)"
```

Comment:
The `doc.go` scares me a little. Would this work? What if other .go files show up in the root dir?
```
ALL_DIRS="$(os::util::list_go_src_files | cut -d '/' -f 1-2 | grep -v ".go$" | grep -v "^./cmd" | LC_ALL=C sort -u)
```